### PR TITLE
You can pair up god eyes

### LINKED ISSUE #19307: If you have an eye of god and pick up another one it gets stuck to your hand
image

I thought i'd be able to combine them

You should be able to combine them so both of your eyes flash or you get some unique power or some shit because the amazing nodrop power kinda sucks ass

### PR DESCRIPTION
**Pull request by Incoming5643**
For the sake of looking cooler/showing off your bad RNG/Freeing up your nodrop hands

Fixes #19307

### CODE REVIEW

**Comment by Incoming5643** [comment]
Don't you ask for a picture, it's a flashing pixel.

**Comment by lzimann** [comment]
GIVE FLASHING PIXELS

File: code/modules/clothing/glasses/glasses.dm [hunk 1]
```diff
@@ -291,6 +291,17 @@
 	flags = NODROP
 	invis_view = SEE_INVISIBLE_MINIMUM
 
+/obj/item/clothing/glasses/godeye/attackby(obj/item/weapon/W as obj, mob/user as mob, params)
+	if(istype(W, src) && W != src && W.loc == user)
+		if(W.icon_state == "godeye")
```
Comment (JJRcop) [inline code/modules/clothing/glasses/glasses.dm]: yuck

Comment (Incoming5643) [inline code/modules/clothing/glasses/glasses.dm]: it's a string, same as any other string. Don't be closeminded.

Comment (JJRcop) [inline code/modules/clothing/glasses/glasses.dm]: no um, you're checking icon_state

**Comment by Core0verload** [comment]
What if someone would find three?

**Comment by Kromgar** [comment]
you shoot rays of disintigration from your eye

File: code/modules/clothing/glasses/glasses.dm (hunk 1 above)
Comment (Incoming5643) [inline code/modules/clothing/glasses/glasses.dm]: It's behind an istype check that limits it to `/godeye` s. Also you're implying that "godeye" is a icon_state that makes sense in any other capacity.

It saves an object var man, an object var!

**Comment by Incoming5643** [comment]
@Core0verload It just says you're really unlucky and you lose the third eye, freeing your hand.

File: code/modules/clothing/glasses/glasses.dm [hunk 2]
```diff
@@ -291,6 +291,17 @@
 	flags = NODROP
 	invis_view = SEE_INVISIBLE_MINIMUM
 
+/obj/item/clothing/glasses/godeye/attackby(obj/item/weapon/W as obj, mob/user as mob, params)
+	if(istype(W, src) && W != src && W.loc == user)
+		if(W.icon_state == "godeye")
+			W.icon_state = "doublegodeye"
+			W.item_state = "doublegodeye"
```
Comment (ChangelingRain) [inline code/modules/clothing/glasses/glasses.dm]: You'll need to update the glasses overlay so you actually have both eyes.

Comment (Incoming5643) [inline code/modules/clothing/glasses/glasses.dm]: It does so, now

**Pull request merged**